### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
I would like to add you as code owners for the respective change logs/release notes sections. Please let me know if this is ok and/or if I should add someone else. We might fine-grain this by adding the name of the build-artifact?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,6 +6,10 @@
 # These owners will be the default owners for everything in the repo.
 * @BeateRixen marlene.windel@softwareag.com
 
+/content/change-logs/application-enablement/ @rahultsag @janhommes @BeateRixen
+/content/change-logs/device-management/ @Oezge-Akin @aaronraab @l2p- @BeateRixen
+/content/change-logs/platform-services/ @airdroplet @martin-sag @kalavathig1 @sarangovindarajan @pink-mongoose @BeateRixen
+
```
Comment (BeateRixen) [inline .github/CODEOWNERS:12]:
```suggestion
/content/change-logs/application-enablement/dtm* @rahultsag @pavanshivajirao  @BeateRixen
```

**Review by rahultsag — APPROVED** [review]

**Review by kalavathig1 — APPROVED** [review]

**Review by Oezge-Akin — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,6 +6,10 @@
 # These owners will be the default owners for everything in the repo.
 * @BeateRixen marlene.windel@softwareag.com
 
+/content/change-logs/application-enablement/ @rahultsag @janhommes @BeateRixen
+/content/change-logs/device-management/ @Oezge-Akin @aaronraab @l2p- @BeateRixen
+/content/change-logs/platform-services/ @airdroplet @martin-sag @kalavathig1 @sarangovindarajan @pink-mongoose @BeateRixen
+/content/change-logs/application-enablement/dtm* @rahultsag @pavanshivajirao  @BeateRixen
 /content/device-management-application/ @Oezge-Akin
```
Comment (Oezge-Akin) [inline .github/CODEOWNERS:13]:
```suggestion
/content/device-management-application/ @Oezge-Akin @aaronraab @l2p- @BeateRixen
```

Comment (BeateRixen) [reply .github/CODEOWNERS:13]: The PMs (Aaron) should actually only review and approve the change logs not all doc changes.

Comment (Oezge-Akin) [reply .github/CODEOWNERS:13]: updated the suggestion

**Comment by BeateRixen** [comment]
I will merge this for now. Please let me know if you want to be removed as code owner or if you want others from your teams to be added.

**Pull request merged**